### ISSUE TITLE
a11y issues in documentation/react/table 

### ISSUE THREAD

**Issue by lareplog** [patternfly/patternfly-react#2737]
for https://github.com/patternfly/patternfly-org/issues/1319 one type of a11y issue needs to be fixed:

1. scope attribute should be used correctly

goes along with this issue https://github.com/patternfly/patternfly-next/issues/2165

**Comment by rachael-phillips** [comment]
Hi @lareplog what was the rationale for tagging this as P1?

**Comment by lareplog** [comment]
@rachael-phillips the original issue was tagged as p1, so i assumed it and the next issue i created would also be considered p1
